ci/generate.go

```diff
@@ -61,3 +66,33 @@ func (m *Generate) WebSdk() *dagger.Directory {
 		Directory("/work/client/web").
 		WithoutDirectory("node_modules")
 }
+
+func (m *Generate) Check(ctx context.Context) error {
+	app := goModuleCross("").
+		WithSource(m.Source).
+		Container().
+		WithEnvVariable("GOFLAGS", "-tags=musl")
+
+	result := app.
+		WithExec([]string{"go", "generate", "-x", "-tags=musl", "-ldflags", "linkmode=external", "./..."}).
+		Directory("")
+
+	source := m.Source
+
+	err := diff(ctx, source, result)
+	if err != nil {
+		return fmt.Errorf("go generate wasn't run: %w", err)
+	}
+	return nil
+}
+
+func diff(ctx context.Context, d1, d2 *dagger.Directory) error {
+	_, err := dag.Container(dagger.ContainerOpts{Platform: ""}).
+		From(alpineBaseImage).
+		WithExec([]string{"apk", "add", "--update", "--no-cache", "ca-certificates", "tzdata", "bash"}).
+		WithDirectory("src", d1).
+		WithDirectory("res", d2).
+		WithExec([]string{"diff", "-u", "-r", "-q", "src", "res"}).
+		Sync(ctx)
```

Comment:
```suggestion
		Sync(ctx)

```

Looks like the linter is disabled for this dir.